bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/BundleProcessor.java

```diff
@@ -121,7 +121,7 @@ CSVMetadataBuildItem gatherCSVMetadata(KubernetesConfig kubernetesConfig,
 
         // note that version, replaces, etc. should probably be settable at the reconciler level
         // use version specified in bundle configuration, if not use the one extracted from the project, if available
-        final var version = kubernetesConfig.getVersion().orElse(appConfiguration.getVersion());
+        final var version = kubernetesConfig.version().orElse(appConfiguration.getVersion());
```

Comment:
Not sure if we should push different changes in the update commits.

Reply:
This is a required change. The code won't compile without this change. This is the reason why the ecosystem checks have been failing for so long as well.

Reply:
OK, thanks for the explanation.